invenio_cli/commands/requirements.py

```diff
@@ -148,22 +170,26 @@ def check_imagemagick_version(cls, major, minor=-1, patch=-1,
     def check_pipenv_installed(cls):
         """Check the pipenv version."""
         # Output comes in the form of 'pipenv, version 2020.11.15\n'
-        result = run_cmd(["pipenv", "--version"])
+        try:
+            result = run_cmd(["pipenv", "--version"])
+
+            parts = result.output.strip().split(',')
+            if parts[0] != 'pipenv':
+                # note that result.error is stderr (type str) of the process
+                raise Exception(result.error)
```

Comment:
It can be the case that pipenv --version returns a string that doesn't start with `pipenv`?

Why not the ProcessResponse here? (if the above is valid)
```
return ProcessResponse(
                error=f"Pipenv not ok. Got {parts[0]}.", status_code=1
            )
```

Reply:
i think that could be the case if somehow `pipenv` points to a different binary, e.g. if it's aliased or symlinked or something... i just kept the original logic here.
regarding returning the `ProcessResponse`, that's probably a better idea than raising an `Exception` b/c it's more legible. will update.

Reply:
> points to a different binary
Sure maybe the original logic was a little too weary :shrug: ...